config/l5-swagger.php

```diff
@@ -45,13 +45,20 @@
 
     /*
     |--------------------------------------------------------------------------
-    | Edit to set the api key variable
+    | Edit to set the api key variable/header name
     |--------------------------------------------------------------------------
     */
     "api-key-var" => env('API_KEY_VAR', 'api_key'),
 
     /*
     |--------------------------------------------------------------------------
+    | Edit to set the securityDefinition that is used in requests
+    |--------------------------------------------------------------------------
+    */
+    "security-definition" => env('SECURITY_DEFINITION', 'apiKey'),
```

Comment:
@TheWildHorse shouldn't ir be 'api_key' by default ? It could mess up older  implementations, because now they are the same ? Correct me if I am wrong.